src/applications/disability-benefits/all-claims/tests/00-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,272 @@
+  PageHelpers.initPaymentInformationMock(token);
+
+  // Ensure introduction page renders.
+  client.assert
+    .title('Apply for disability benefits: VA.gov')
+    // First render of React may be slow.
+    .waitForElementVisible('.schemaform-title', Timeouts.slow) // First render of React may be slow.
+    .waitForElementVisible(
+      '.schemaform-intro .usa-button-primary',
+      Timeouts.verySlow,
+    )
+    .click('.schemaform-intro .usa-button-primary')
+    // Click past the `You already have an Intent to File` screen.
+    .waitForElementVisible('.usa-grid .usa-button-primary', Timeouts.slow)
+    .click('.usa-grid .usa-button-primary');
+
+  E2eHelpers.overrideVetsGovApi(client);
+  FormsTestHelpers.overrideFormsScrolling(client);
+  E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
+
+  // Veteran Details
+  // Review Veteran Information
+  E2eHelpers.expectLocation(client, '/veteran-information');
+  client.axeCheck('.main');
+  client.click('.form-progress-buttons .usa-button-primary');
+
+  // Alternate Name
+  E2eHelpers.expectLocation(client, '/alternate-names');
+  client.axeCheck('.main');
+  PageHelpers.completeAlternateName(client, testData.data);
+  client.click('.form-progress-buttons .usa-button-primary');
+
+  // Military Retirement Pay
+  E2eHelpers.expectLocation(client, '/service-pay');
+  client.axeCheck('.main');
+  PageHelpers.completeMilitaryRetiredPay(client, testData.data);
+  client.click('.form-progress-buttons .usa-button-primary');
+
+  // Military Service History
+  E2eHelpers.expectLocation(client, 'military-service-history');
+  client.axeCheck('.main');
+  PageHelpers.completeMilitaryHistory(client, testData.data);
+  client.click('.form-progress-buttons .usa-button-primary');
+
+  // Combat Zone Post 9/11
+  E2eHelpers.expectLocation(client, '/combat-status');
+  client.axeCheck('.main');
+  PageHelpers.completeCombatZonePost911(client, testData.data);
+  client.click('.form-progress-buttons .usa-button-primary');
+
+  // Reserves/National Guard Info
+  E2eHelpers.expectLocation(client, 'reserves-national-guard');
+  client.axeCheck('.main');
+  PageHelpers.completeReservesNationalGuardInfo(client, testData.data);
+  client.click('.form-progress-buttons .usa-button-primary');
+
+  // Federal Orders
+  E2eHelpers.expectLocation(client, '/federal-orders');
+  client.axeCheck('.main');
+  PageHelpers.completeFederalOrders(client, testData.data);
+  client.click('.form-progress-buttons .usa-button-primary');
+
+  // Disabilities
+  // Orientation
+  E2eHelpers.expectLocation(client, '/disabilities/orientation');
+  client.axeCheck('.main').click('.form-progress-buttons .usa-button-primary');
+
+  // Rated Disability Selection
+  E2eHelpers.expectLocation(client, '/disabilities/rated-disabilities');
+  client.axeCheck('.main');
+  PageHelpers.selectDisabilities(client); // Just selects the first one
+  client.click('.form-progress-buttons .usa-button-primary');
+  // ****
```

Comment:
All tests below are commented out as the flow of all-claims is currently in flux. They worked for the page/form they are for before the flow before them was changed.